CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.7.0 - 2021-05-12
+### Added
+- Project validation prior to `Create Static Web App...`
```

Comment:
I would add something about git. This makes me think we validate their app before creating
```suggestion
- Git project validation prior to `Create Static Web App...`
```